packages/ui/src/components/Widgets/AuthorizedStudies/index.tsx

```diff
@@ -4,34 +4,55 @@ import { Button, List, Result, Space } from 'antd';
 import Text from 'antd/lib/typography/Text';
 import cx from 'classnames';
 
-import GridCard, { GridCardHeader } from '../../view/v2/GridCard';
+import GridCard, { GridCardHeader } from '../../../view/v2/GridCard';
 import CardConnectPlaceholder, {
+    DEFAULT_CARD_CONNECT_PLADHOLDER_DICTIONARY,
     TCardConnectPlaceholderDictionary,
-} from '../../view/v2/GridCard/GridCardConnectPlaceholder';
-import Empty from '../Empty';
-import ExternalLink from '../ExternalLink';
-import PopoverContentLink from '../PopoverContentLink';
+} from '../../../view/v2/GridCard/GridCardConnectPlaceholder';
+import Empty from '../../Empty';
+import ExternalLink from '../../ExternalLink';
+import PopoverContentLink from '../../PopoverContentLink';
+import { WIDGET_STATE } from '../constants';
 
 import AuthorizedStudiesListItem, {
+    DEFAULT_AUTHORIZED_STUDIES_LIST_ITEM_DICTIONARY,
     IAuthorizedStudyQueryProps,
     TAuthorizedStudiesListItemDictionary,
 } from './AuthorizedStudiesListItem';
-import FencesAuthentificationModal, { TFencesAuthentificationModalDictionary } from './FencesAuthentificationModal';
+import FencesAuthentificationModal, {
+    DEFAULT_FENCES_AUTHENTIFICATION_MODAL_DICTIONARY,
+    TFencesAuthentificationModalDictionary,
+} from './FencesAuthentificationModal';
 
-import styles from './index.module.scss';
-
-enum WIDGET_STATE {
-    error = 'error',
-    connected = 'connected',
-    disconnected = 'disconnected',
-}
+import styles from '../widget.module.scss';
 
 export enum FENCE_AUHTENTIFICATION_STATUS {
```

Comment:
```suggestion
export enum FENCE_AUTHENTIFICATION_STATUS {
```
Et tous les imports et utilisation dans les autres fichiers.